en/docs/administer/managing-users-and-roles/managing-user-stores/configure-primary-user-store/configuring-a-read-write-ldap-user-store.md

```diff
@@ -191,23 +501,39 @@ The configuration for the external read/write user store in the `user-mgt.xml` f
 
 ### Step 2: Updating the system administrator
 
-The **admin** user is the super tenant that will be able to manage all other users, roles and permissions in the system by using the management console of the product. Therefore, the user that should have admin permissions is required to be stored in the user store when you start the system for the first time. Since the LDAP user store can be written to, you have the option of creating a new admin user in the user store when you start the system for the first time. Alternatively, you can also use a user ID that already exists in the LDAP. For information about the system administrator user, see [Configuring the System Administrator]({{base_path}}/reference/config-catalog/#super-admin-configurations) .
+The **admin** user is the super tenant that will be able to manage all
+other users, roles, and permissions in the system by using the management
+console of the product. Therefore, the user that should have admin
+permissions is required to be stored in the userstore when you start
+the system for the first time. Since the LDAP userstore can be written
+to, you have the option of creating a new admin user in the userstore
+when you start the system for the first time. Alternatively, you can
+also use a user ID that already exists in the LDAP. For information
+about the system administrator user, see [Configuring the System
+Administrator]({{base_path}}/reference/config-catalog/#super-admin-configurations) .
```

Comment:
Any reason to not have this in a single paragraph?

Reply:
fixed the formatting